notebooks/running_jobs.ipynb

```diff
@@ -89,11 +127,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
-   "outputs": [],
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "FileNotFoundError",
```

Comment:
looks like this blows up because sample_config.txt is now sample_config_monomer.txt